modules/apps/frontend-js/frontend-js-importmaps-extender/src/main/java/com/liferay/frontend/js/importmaps/extender/internal/servlet/taglib/JSImportMapsExtenderTopHeadDynamicInclude.java

```diff
@@ -65,8 +59,7 @@ public void include(
 		PrintWriter printWriter = httpServletResponse.getWriter();
 
 		if (_jsImportMapsConfiguration.enableImportMaps() &&
-			(!_globalImportMapJSONObjects.isEmpty() ||
-			 !_scopedImportMapJSONObjects.isEmpty())) {
+			JSImportMapsRegistryUtil.isBothNotEmptyImportMapsJSONObjects()) {
```

Comment:
@MrJohn1911, rename this to `isImportMapsJSONObjectsEmpty()` instead and apply the required changes. Thx